R/scholar.r

```diff
@@ -252,3 +252,86 @@ plot_coauthors <- function(network, size_labels = 5) {
     ggplot2::labs(title = paste0("Network of coauthorship of ", network$author[1])) +
     ggraph::theme_graph(title_size = 16, base_family = "sans")
 }
+
+
+
+
+
+
+
+
+##' Get author order.
+##'
+##' Get author rank in authors list.
+##'
+##' @examples
+##' library(scholar)
+##'
+##' id <- "bg0BZ-QAAAAJ&hl"
+##'
+##' publications <- scholar::get_publications(id)
+##' publications <- publications$author
```

Comment:
I think this overwriting of variables, especially given slightly confusing argument names of function is best avoided.

Reply:
Corrected